lib/src/view/design_main_strand_moving.dart

```diff
@@ -135,8 +135,12 @@ class DesignMainStrandMovingComponent extends UiComponent2<DesignMainStrandMovin
           }
           helix_svg_position_y = props.helix_idx_to_svg_position_map[helix.idx].y;
           start_svg = helix.svg_base_pos(domain.offset_5p, domain.forward, helix_svg_position_y);
-          var control = control_point_for_crossover_bezier_curve(old_domain, domain, props.helices,
-              props.helix_idx_to_svg_position_map[old_domain.helix].y, props.helix_idx_to_svg_position_map[domain.helix].y,
+          var control = control_point_for_crossover_bezier_curve(
```

Comment:
It would be really great to handle issue #777. Most of the changes to files in this PR are formatting changes. Once #777 is implemented we'll all have the same dart format rules, so changes in PR's will then only be logic changes.